.circleci/config.yml

```diff
@@ -280,7 +283,7 @@ jobs:
     steps:
       - integration-tests:
           local: true
-          env: local
+          env: staging
   integration-tests-alpha:
     executor: puppeteer
     parallelism: 1 # No way to prevent multiple Slack messages otherwise
```

Comment:
Unrelated: Comment is not true anymore. We don't use Slack orb to send Slack anymore. Can you remove it in this PR?

Reply:
Good catch! Will probably prevent a lot of confused heads in the future 😊

Reply:
If this is no longer true, why do we still use `parallelism: 1` here?

Reply:
> If this is no longer true, why do we still use parallelism: 1 here?

Good question. Might be worth experimenting with a higher number.